README.md

```diff
@@ -1,19 +1,29 @@
+Extension for [openid-client](https://www.npmjs.com/package/openid-client) to assist with the full [OAuth 2.0](https://tools.ietf.org/html/rfc6749) [Client](https://tools.ietf.org/html/rfc6749#section-1.1) or [OpenID Connect](https://openid.net/connect/) (OIDC) [Relying Party](https://openid.net/specs/openid-connect-core-1_0.html#Terminology) (RP) implementation and token management.
 
 ## Motivation
 
-The openid-client library provides components for building a [Node.js](https://nodejs.org/en/) application playing the role of an OIDC [Relying Party](https://openid.net/specs/openid-connect-core-1_0.html#Terminology) (RP) and includes [Passport.js](http://www.passportjs.org/) authentication strategy middleware. The means of performing authorization need to be build and then executed for each protected resource.
+The openid-client library provides components for building [Node.js](https://nodejs.org/en/) applications playing the role of an RP. The library includes [Passport.js](http://www.passportjs.org/) authentication strategy middleware. The means of performing authorization need to be build and then executed for each protected resource.
 
-The openid-client-helper package allows for performing a single [OAuth 2.0](https://tools.ietf.org/html/rfc6749) authorization grant and maintaining separate authorization states for multiple resources protected by the same [Authorization Server](https://tools.ietf.org/html/rfc6749#section-1.1).
+The openid-client-helper package allows for performing a single OAuth 2.0 authorization grant and maintaining separate authorization states for multiple resources protected by the same [Authorization Server](https://tools.ietf.org/html/rfc6749#section-1.1).
 
 The authorization can be performed with the helper express middleware or with external means, like the Passport.js strategy. If the resulting token set contains refresh token, it can be used to obtain access tokens for individual resources. The helper exposes the redefined [node-fetch](https://www.npmjs.com/package/node-fetch#difference-from-client-side-fetch) `fetch` method for making HTTP requests to the resources protected by OAuth 2.0. Each individual request will be crafted with an access token specific to the requested resource. The token will be automatically obtained and, if necessary, renewed while handling the request.
 
 This means that after initial authorization by the [Resource Owner](https://tools.ietf.org/html/rfc6749#section-1.1) (RO), the RP will be able to perform API requests with [Audience Restricted Access Tokens](https://tools.ietf.org/html/draft-ietf-oauth-security-topics-13#section-4.8.1.3) according to the best current practices (BCPs), performing authorization for individual resources entirely via the back-channel; that is, without further RO involvement.
 
 ## Features
 
+* Authorization and authentication middleware
+* Deauthorization and RP initiated logout middleware
+* Audience restricted access tokens
```

Comment:
Wondering: audience-restricted  ?

Reply:
This one, I will actually comment on: I like the hyphen connected version better as well, but that's not what they do in the original spec (https://tools.ietf.org/html/draft-ietf-oauth-security-topics-13#section-4.8.1.3), so I followed.

Reply:
That is absolutely correct. :)